### DISCUSSION TITLE
Cloudflare DDNS auto implement trough API?

### DISCUSSION THREAD

**Discussion by clemone210** [qdm12/ddns-updater#318] [Ideas]
I was wondering if it would be possible to fetch all the websites from Cloudflare and their respective DNS entries and then auto update the config?
Isnt there an API which could be used?
Would that be possible in some way?

**Comment by qdm12** [comment]
Yes but they are cases where you don't want to update all your records. But yes there is most likely API calls available to list all the records of a domain etc.
